docs/Intro/dive/offchain.md

```diff
@@ -3,15 +3,19 @@ sidebar_position: 2
 description: Onchain components of Chronicle Protocol Architecture.
 keywords: [data sources, offchain, Validators, Challenger]
 ---
-# Off-Chain Components 
 
-    - **Validator** (aka Feed) Creates usable information out of raw data that is read from various Origins. Validators query Origins no matter whether they are on-chain or off-chain for the price data. The Validator transform the original data into Chronicle data by using Data Models. After the Chronicle data is calculated it is signed and sent to the peer-to-peer network.
+# Offchain Components
+
+     - **Validators** (aka Feeds) interact with the peer-to-peer Validator network to read and write messages, and they also read data from relays. Validators process this raw data into usable information using different Data Models, transforming it into Chronicle data (prices for different asset pairs). Once calculated, this data is signed and sent to the peer-to-peer network.
```

Comment:
Vlidators DO NOT read anything from relays.

Also, I am not sure if we want to keep "Feeds" - it might confuse people - but that is a secondary issue.

Validators read data from different sources (origins - i.e. where the data originates from) (both on-chain and off-chain). Then they apply a data model to that so as to calculate a final price that is resistant to market manipulation.

("Validators process this raw data into usable information using different Data Models" sounds very nice)

*...Once calculated, this data is signed and sent to the peer-to-peer network.*

Validators interact with each other during Schnorr signing session.